### PR TITLE
Ci refactoring

### PR DESCRIPTION
**Pull request by AndreyLykhoman**
**Description:** 
Changes:
 - Reorganization docker files 
 - Reorganization docker-compose files
 - Add docker-compose and comands in make files for selenium tests
 - Reorganization .travis.yml for run tests on CI with the separeted stages. Prepare to add the selenium stage
 - Create Makefile files with make commands for convenient to update/run server and run tests
 - Change the update_stage_script.py. Use make command

This optimizations make build and run Docker images quickly and prepare to add selenium tests with convenient to use this and change settings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,15 +57,14 @@ Before running deployment configure `secure.py` settings in the
...
 
-    [sudo] docker-compose -f docker-compose_local.yml up
+    [sudo] make run-local
 
+Volume "pgs" is added to the the database container.
 
-  Volume "pgs" is added to the the database container.
-
-  Note: Development server available on `localhost:8008`
+  Note: Development server available on `localhost:8000`
```
Comment (idegtiarov) [inline README.md]: This change looks odd to me

File: bridge_adaptivity/nginx/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
 
 RUN apt-get update && apt-get install -y vim
+RUN rm /etc/nginx/conf.d/default.conf
+
+ARG build_env=prod
+ADD $build_env/* /etc/nginx/conf.d/
+
+RUN mkdir /etc/nginx/ssl
+
+COPY --from=bridge /www/static /www/static
```
Comment (idegtiarov) [inline bridge_adaptivity/nginx/Dockerfile]: trailing empty line is missed

File: bridge_adaptivity/prod_run.sh [hunk 3]
```diff
@@ -1,5 +0,0 @@
-#!/usr/bin/env bash
-
-python manage.py migrate --settings=config.settings.prod
-python manage.py collectstatic -v 3 -c --noinput --settings=config.settings.prod
```
Comment (idegtiarov) [inline bridge_adaptivity/prod_run.sh]: Could you please point me on which step do we collect static within current changes?

File: bridge_adaptivity/docker-compose-stage.yml [hunk 4]
```diff
@@ -1,62 +1,8 @@
...
-    command: bash -c "./prod_run.sh"
-    volumes:
-      - .:/bridge_adaptivity
-      - static:/www/static
-    ports:
-      - "8000:8000"
-    links:
-      - postgres
-  # Celery worker
-  worker:
```
Comment (idegtiarov) [inline bridge_adaptivity/docker-compose-stage.yml]: Clarify please why worker and rabbit are removed from the stage docker-compose file?

File: AUTHORS [hunk 5]
```diff
@@ -1,6 +1,11 @@
+Alex <django.develop@gmail.com>
```
[the remark this answers is not in the document]
Comment (AndreyLykhoman) [reply AUTHORS]: I deleted this.

File: README.md (hunk 1 above)
Comment (AndreyLykhoman) [reply README.md]: I deleted this change. Local Bridge run in the 8008 port.

File: bridge_adaptivity/nginx/Dockerfile (hunk 2 above)
Comment (AndreyLykhoman) [reply bridge_adaptivity/nginx/Dockerfile]: I added this line.

File: bridge_adaptivity/prod_run.sh (hunk 3 above)
Comment (AndreyLykhoman) [reply bridge_adaptivity/prod_run.sh]: Static is collected when we build new images. You can see this step in the Dockerfile: https://github.com/harvard-vpal/bridge-adaptivity/pull/233/files#diff-6da744b8e40a699166501279107d566cR37

File: bridge_adaptivity/docker-compose-stage.yml (hunk 4 above)
Comment (AndreyLykhoman) [reply bridge_adaptivity/docker-compose-stage.yml]: This version of configuring and running Bridge has the make commands for run prod (`make run`), stage (`make run-stage`), and local (`make run-local`) version.  For stage the make command looks like: 
`docker-compose -f docker-compose.yml -f docker-compose-stage.yml up -d`

The prod docker-compose has the main configuration. The stage docker-compose has a specific configuration for the stage. If you run this command: 
`docker-compose -f docker-compose.yml -f docker-compose-stage.yml config`
You can see the docker-compose configuration for the stage. And this configuration has the rabbit and the worker configuration.

File: bridge_adaptivity/prod_run.sh (hunk 3 above)
Comment (AndreyLykhoman) [reply bridge_adaptivity/prod_run.sh]: I added collecting static in the local machine when it's starting.
it's script:
https://github.com/harvard-vpal/bridge-adaptivity/pull/233/files#diff-6e3f0ef6ecb83cb1a9a0931ed82e5c4bR2
in this line it is used: 
https://github.com/harvard-vpal/bridge-adaptivity/pull/233/files#diff-6da744b8e40a699166501279107d566cR46

**Comment by idegtiarov** [comment]
I've added some comments in another PR. Please take a look at them.

I would do PRs so that the code isn't mixed between them. So review process would become much simpler.

File: bridge_adaptivity/config/settings/base.py [hunk 6]
```diff
@@ -192,3 +193,14 @@
...
+ASGI_APPLICATION = "config.routing.application"
+
+CHANNEL_LAYERS = {
+    "default": {
+        "BACKEND": "channels_redis.core.RedisChannelLayer",
+        "CONFIG": {
+            "hosts": [("redis", 6379)],
+        },
+    },
+}
```
Comment (idegtiarov) [inline bridge_adaptivity/config/settings/base.py]: A good practice is to add a trailing empty line at the end of a file.

Comment (AndreyLykhoman) [reply bridge_adaptivity/config/settings/base.py]: I added the new line.

**Pull request merged**